### PR TITLE
backport some strings to stable7

### PR DESCRIPTION
**Pull request by vgezer**
supersedes #12322 and #12324

@DeepDiver1975 @MorrisJobke

### CODE REVIEW

**Comment by MorrisJobke** [comment]
@owncloud-bot retest this please

**Comment by MorrisJobke** [comment]
:+1:

**Comment by MorrisJobke** [comment]
@DeepDiver1975 Want to have a look at this? ;)

**Comment by MorrisJobke** [comment]
@DeepDiver1975 ping

**Comment by wakeup** [comment]
What do we do with those strings? Do we still merge them?

**Comment by MorrisJobke** [comment]
@DeepDiver1975 merge it or kill it

File: core/l10n/az.php [hunk 1]
```diff
@@ -4,6 +4,7 @@
 "_%n hour ago_::_%n hours ago_" => array("",""),
 "_%n day ago_::_%n days ago_" => array("",""),
 "_%n month ago_::_%n months ago_" => array("",""),
-"_{count} file conflict_::_{count} file conflicts_" => array("","")
+"_{count} file conflict_::_{count} file conflicts_" => array("",""),
```
Comment (DeepDiver1975) [inline core/l10n/az.php]: @nickvergessen which problem? The was an issues - yes .... but I don't remember ...

**Comment by DeepDiver1975** [comment]
:+1:

File: core/l10n/az.php (hunk 1 above)
Comment (nickvergessen) [inline core/l10n/az.php]: The translation does not fallback to the english version:
https://github.com/owncloud/core/pull/12779

Comment (nickvergessen) [inline core/l10n/az.php]: phpStorm found 889 cases of those empty arrays which will result in empty strings being displayed in stable7 :/

Comment (DeepDiver1975) [inline core/l10n/az.php]: ok - looks like we should backport https://github.com/owncloud/core/pull/12779 - can I ask you to prepare a PR? THX

Comment (nickvergessen) [inline core/l10n/az.php]: Done: #14297

**Pull request merged**